client/src/components/notifications/Notification.tsx

```diff
@@ -1,30 +1,27 @@
-import { Toaster } from "react-hot-toast";
+import { Toaster, ToastOptions } from "react-hot-toast";
 
-export const SUCCESS = {
+const POSITIVE: ToastOptions = {
   duration: 4000,
   position: "bottom-right",
-  // Styling
   style: {
     backgroundColor: "#0E9860",
     color: "white",
     fontWeight: "bold",
-    borderRadius: "3px",
+    borderRadius: "0.2rem",
```

Comment:
Ensured `borderRadius`, `duration` and `position` were the same for all options.